test/PeNet.Test/Header/Resource/ResourcesTest.cs

```diff
@@ -203,21 +203,10 @@ public void IconsAsRaw_GivenPeFilesAndIconIncludedInPeFile_FoundIconContainsGive
             foreach (var iconFile in Directory.EnumerateFiles(iconDirectory))
             {
                 var icon = File.ReadAllBytes(iconFile);
-                Assert.Contains(icon, peFile.IconsAsRaw());
+                Assert.Contains(icon, peFile.Icons());
```

Comment:
This test should fail now, or does this check for partial arrays?

Reply:
Similar test to test in line 238.
(IconsAsICO_GivenPeFilesAndIconIncludedInPeFile_FoundIconContainsGivenIcon)
[can be removed like IconsAsRaw_GivenPeFilesAndIconNotIncludedInPeFile_FoundIconDoNotContainsGivenIcon]

Reply:
Why should the test fail?

Reply:
If I understand it correctly, we load byte arrays without headers (.raw) and test whether they are included in the list of icons with headers (.ico) or not.